docs/contribute/style-guide.md

```diff
@@ -0,0 +1,182 @@
+---
+---
+
+{% capture overview %}
+This page gives writing style guidelines for the Kubernetes documentation.  These are guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+For additional information on creating new content for Kubernetes, follow the instructions on [using page template](/docs/contribute/page-templates/) and creating a [creating a documentation pull request](/docs/contribute/create-pull-request/).
+{% endcapture %}
+
+{% capture body %}
+
+## Documentation formatting standards
+
+The following sections detail how to format API objects, code snippets, 
```

Comment:
Incomplete sentence?

I'd also make it clearer: "The following sections detail how to refer to Kubernetes API objects outside of code snippets, e.g. in paragraphs."